src/utils/DepositUtils.ts

```diff
@@ -208,6 +208,11 @@ export function isZeroValueDeposit(deposit: Pick<Deposit, "inputAmount" | "messa
   return deposit.inputAmount.eq(0) && isMessageEmpty(deposit.message);
 }
 
+export function invalidOutputToken(deposit: Pick<DepositWithBlock, "outputToken">): boolean {
+  // If the output token is zero address, then it is invalid.
+  return deposit.outputToken === ZERO_ADDRESS;
```

Comment:
Should this be `ZERO_BYTES` ? I think fwiw we already have a similar function (`isZeroAddress()`) used by the SpokePoolClient to determine whether it needs to sub in a valid address.

Reply:
https://github.com/across-protocol/sdk/pull/970/commits/cfabc9b1348c69b6faa5046898b4e8e15be211eb